riptide-chaos/src/main/java/org/zalando/riptide/chaos/EmptyInputStream.java

```diff
@@ -0,0 +1,19 @@
+package org.zalando.riptide.chaos;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+final class EmptyInputStream extends InputStream {
```

Comment:
this incarnation of this class is untested :stuck_out_tongue_closed_eyes:

Reply:
How do you know?

Reply:
I only see a test for the one of `riptide-httpclient`... but I also see the similarities between the implementations.